### ISSUE TITLE
FOI: might soon need to remove my recent ENH for github authentication...

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#3947]
Used create-sibling-github today and later received email:
```shell
Date: Wed, 18 Dec 2019 12:46:06 -0800                                                                                                                                                                                              
From: GitHub <noreply@github.com>                                                                                                                                                                                                  
To: Yaroslav Halchenko <debian@onerussian.com>                                                                                                                                                                                     
Subject: [GitHub] Deprecation Notice                                                                                                                                                                                               
...                                                                                                                                                                                                                                   
                                                                                                                                                                                      
You recently used a password to access an endpoint through the GitHub API using PyGithub/Python. We will deprecate basic authentication using password to this endpoint soon:                                                      
                                                                                                                                                                                                                                   
https://api.github.com/user                                                                                                                                                                                                        
                                                                                                                                                                                                                                   
We recommend using a personal access token (PAT) with the appropriate scope to access this endpoint instead. Visit https://github.com/settings/tokens for more information.                                                        
                                                                                                                                                                                                                                   
Thanks,                                                                                                                                                                                                                            
The GitHub Team 
```
which is I guess due to us obtaining a token programmatically using https://pypi.python.org/pypi/PyGithub , not sure if there is anything it should do on its end I have [not spotted any related issue related to such an email in their issue tracker](https://github.com/PyGithub/PyGithub/search?q=endpoint+through+the+GitHub+API+using+PyGithub%2FPython&type=Issues)

**Comment by yarikoptic** [comment]
well, it is time now since 
```
github.GithubException.GithubException: 401 {'message': "Bad credentials. The API can't be accessed using username/password authentication. Please create a personal access token to access this endpoint: http://github.com/settings/tokens", 'documentation_url': 'https://docs.github.com/articles/creating-a-personal-access-token-for-the-command-line'}
```
so we need to strip away minting of tokens, which could also open possibility to simplify logic there.

**Comment by yarikoptic** [comment]
I believe it was addressed by #5218 and already released
